Components/GameBoard/ActivePlayerPrompt.jsx

```diff
@@ -107,7 +110,7 @@ class ActivePlayerPrompt extends React.Component {
         let timer = null;
 
         let promptText = [];
-        if(this.props.promptText.includes('\n')) {
+        if(this.props.promptText && this.props.promptText.includes('\n')) {
```

Comment:
Fixed it this was in keyteki too, but it needs properly fixing, spectators should get some sort of prompt text!